pkgs/servers/vouch-proxy/default.nix

```diff
@@ -0,0 +1,39 @@
+{ pkgs, ... }:
+
+pkgs.buildGoModule rec {
+  pname = "vouch-proxy";
+  version = "0.23.1";
+
+  nativeBuildInputs = [ pkgs.makeWrapper ];
+
+  src = pkgs.fetchFromGitHub {
+    owner = "vouch";
+    repo = "vouch-proxy";
+    rev = "v0.23.1";
+    sha256 = "00nraynyscaxns3c6lpwq7pscflhj87yzy6474biqbp04qyn3m36";
+  };
+
+  vendorSha256 = "1yiagqg0x6n8mail0zz4400bpjvz49kbncm5km2nhapx6znzvghm";
+
+  doCheck = false;
```

Comment:
Please add a comment why checks are disabled.

Reply:
I believe the issue is that their tests expect to be invoked by some non-standard wrapper scripts, which the nixpkgs builders knows nothing about.  It probably could be worked around with sufficient effort.